Formula/c/certgraph.rb

```diff
@@ -25,7 +25,7 @@ def install
   end
 
   test do
-    output = shell_output("#{bin}/certgraph example.com")
+    output = shell_output("#{bin}/certgraph www.example.com")
```

Comment:
We can see the SAN here, https://www.geocerts.com/ssl-checker and so this line is not a problem but the below two are.

Reply:
The problem with my adjusted test is also that both lines below will match the "www.example.org" requested in the CLI parameter.

Reply:
Scratch that, i didn't even notice that example.com, .edu and .org were all in the same certificate, so i found one from github